pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -2188,5 +2189,66 @@ protected void internalSetIsAllowAutoUpdateSchema(boolean isAllowAutoUpdateSchem
         }
     }
 
+    protected void internalSetOffloadPolicies(OffloadPolicies offloadPolicies) {
```

Comment:
nit: ideally we should try to implement this in an asynchronous way.

Reply:
ok, I'll fix it.